install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -3028,6 +3028,13 @@ clustermesh:
       # Allows to configure a specific loadBalancerIP on the clustermesh-apiserver
       # LB service in case the Service type is set to LoadBalancer.
       loadBalancerIP: ~
+      # @schema
+      # type: [array]
+      # @schema
```

Comment:
Nit. Looks like this is inferred automatically, and could be omitted.

Reply:
Cool, removed :)